Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -911,6 +921,7 @@ [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? subtitle
     /// <param name="context">Optional. The <see cref="EncodingContext"/>.</param>
     /// <param name="streamOptions">Optional. The streaming options.</param>
     /// <param name="enableAudioVbrEncoding">Optional. Whether to enable Audio Encoding.</param>
+    /// <param name="alwaysBurnInSubtitleWhenTranscoding">Whether to always burn in subtitles when transcoding.</param>
```

Comment:
Then remove this one as well.
_I expected you to remove them all._ :sweat_smile: 
```suggestion
```